tensorboard/webapp/metrics/views/card_renderer/scalar_card_data_table.ts

```diff
@@ -216,3 +221,12 @@ function getSortableValue(value: number | string | undefined | null) {
   }
   return value;
 }
+
+function getSortableValue(point: SelectedStepRunData, header: ColumnHeaders) {
+  switch (header) {
+    case ColumnHeaders.RUN:
+      return makeValueSortable(point[ColumnHeaders.DISPLAY_NAME]);
```

Comment:
Can you add some tests?

Reply:
I've gone ahead and added tests to the newly refactored version.